contributor-docs/adrs/adr-005.md

```diff
@@ -0,0 +1,65 @@
+# ADR 5: Migration plan for deprecated components
+
+## Status
+
+Proposed
+
+## Context
+
+As we work on maturity of our components, we will sometimes need to deprecate components that are replaced with new components.
+
+## Here are the 3 proposed stages:
+
+### Stage 1
+
+Start new component outside the main bundle. Folks who want to try it, need to explicitly import it from the `drafts` bundle.
```

Comment:
Should drafts be listed in the docs sidenav?

Reply:
yep! 💯

Reply:
For easier discovery, could we consider co-locating them somehow to the original component? Not sure how, maybe tabs? But would help reduce IA complexity too.

Reply:
See updated suggestions! (I think we should not collocate them at all)